recipes/newrelic/nginx/debian.yml

```diff
@@ -16,24 +16,25 @@ keywords:
   - Integration
   - nginx
 
-# Examine Infrastructure events for correlated data
+# CLI runs process detection; this is used to filter recipes that are appropriate for matched processes
 processMatch:
-  - /nginx/
+  - nginx
 
-# Examine Metrics, Events, and Logging for correlated data
-# Used by the UI to determine if you've successfully configured and are ingesting data
-meltMatch:
-  logging:
-    pattern:
-      - /http/
-    files:
-      - /var/log/nginx/
+# Matches partial list of the Log forwarding parameters
+logMatch:
+  - name: nginx
+    file: /var/log/nginx/access.log*  # assumes log rotation like access.log.0, access.log.1, etc.
```

Comment:
Can we just specify the folder instead of all the files in the folder? `file: /var/log/nginx`

Reply:
This is specifically targeted at access logs because of the use of `logtype: nginx` for ruleset parsing. Likewise there's another logMatch entry under this for `nginx-error`.